polygon-zkevm/eth_blockNumber.yaml

```diff
@@ -17,7 +17,26 @@ paths:
       operationId: eth-blocknumber-polygon-zkevm
       $ref: ../ethereum/eth_blockNumber.yaml#/components/pathItems/path/post
       x-readme:
+        explorer-enable: false
```

Comment:
@opman129 not sure if these polygon changes were intentionally included in this PR?